build.env

```diff
@@ -43,7 +43,7 @@ CHANGE_MINIKUBE_NONE_USER=true
 # Rook options
 ROOK_VERSION=v1.4.9
 # Provide ceph image path
-ROOK_CEPH_CLUSTER_IMAGE=docker.io/ceph/ceph:v14.2.12
+ROOK_CEPH_CLUSTER_IMAGE=docker.io/ceph/ceph:v16
```

Comment:
duplicate commit of #2038 ?

Reply:
Yes and this is intentional for satisfying the dependency for this PR at CI. I will remove it once the other PR is merged.